src/cryptography/hazmat/__init__.py

```diff
@@ -1,5 +1,10 @@
 # This file is dual licensed under the terms of the Apache License, Version
 # 2.0, and the BSD License. See the LICENSE file in the root of this repository
 # for complete details.
+"""Hazardous Materials
 
+You should ONLY use it if you're 100% absolutely sure that you know
+what you're doing because this module is full of land mines, dragons,
+and dinosaurs with laser guns.
```

Comment:
I think we want to use the exact originial text here, otherwise the `it` in the first line is an ambigious reference:

```
This is a "Hazardous Materials" module. You should **ONLY** use it if you're
100% absolutely sure that you know what you're doing because this module is
full of land mines, dragons, and dinosaurs with laser guns.
```